packages/@react-types/table/src/index.d.ts

```diff
@@ -14,6 +14,13 @@ import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, LoadingS
 import {GridCollection, GridNode} from '@react-types/grid';
 import {Key, ReactElement, ReactNode} from 'react';
 
+/** Widths that result in a constant pixel value for the same Table width. */
+export type ColumnStaticSize = number | `${number}` | `${number}%`; // match regex: /^(\d+)(?=%$)/
+/** Widths that change size in relation to the remaining space and in ratio to other dynamic columns. */
+export type ColumnDynamicSize = `${number}fr`; // match regex: /^(\d+)(?=fr$)/
+/** All possible sizes a column can be assigned. */
+export type ColumnSize = ColumnStaticSize | ColumnDynamicSize;
```

Comment:
Just a heads up, I noticed that these types were malformed in the docs.
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/9661127/207406204-a4891899-4759-41ad-9fd4-da3301a54648.png">
Looks like it is having a hard time understanding ``${number}` | `${number}%`` and ``${number}fr`` unfortunately

Reply:
urgh, lol, i'll address it in followup
TODO: add to follow up